src/api/app.ts

```diff
@@ -112,16 +112,30 @@ export function create(
         streamServerEvents(req, res, logsSource, (item) => item.toString());
     });
 
+    const uiBaseHref = (req: express.Request) => {
```

Comment:
I think `kubectl proxy` itself update href value for index.html. I've tried to revert changes in app.ts and argo ui still works with `kubectl proxy`. Please revert changes in app.ts (unless I'm missing something)

Reply:
Definitely me missing something :-)

I've reverted changes in app.tsx except for the `{index: false}` option to `express.static` which is necessary to use the modified index HTML when `/` is accessed directly (a6a03b20636e0edc0aa5322c0d065a59ae40d4ae).